.circleci/config.yml

```diff
@@ -30,18 +30,28 @@ jobs:
           keys:
             - composer<< parameters.cv >>-{{ checksum "composer.lock" }}
             - composer<< parameters.cv >>-
-      - run: COMPOSER_MEMORY_LIMIT=-1 composer update
-      - run: composer install -n --prefer-dist
+      - run:
+          name: "Update Composer"
+          command: COMPOSER_MEMORY_LIMIT=-1 composer update
```

Comment:
you have a `composer.lock` file - and ignore it at all by simply using just the latest version instead of the one that was pinned to.

why?

Reply:
Thanks for pointing this out @jkowalleck - fix in PR #39